website/src/App.tsx

```diff
@@ -152,9 +152,15 @@ function App() {
             />
         </ErrorBoundary>
         <div className={Styles.json}>
-            <h1>Message builder</h1>
-            <p style={{marginBottom: '2rem', marginTop: '1rem'}}>Warning: Non-link buttons and select menus are not
-                allowed when sending message via webhook.</p>
+            <h1>discord.builders — Best webhook tool for Discord</h1>
+            <a href="https://github.com/StartITBot/discord.builders" target="_blank"><div className={Styles.badges}>
+                <img alt="Star on GitHub"
+                     src="https://img.shields.io/github/stars/StartITBot/discord.builders?style=for-the-badge&logo=github&label=Star+on+GitHub&color=007ec6" />
+                <img alt="GitHub contributors"
+                     src="https://img.shields.io/github/contributors/StartITBot/discord.builders?style=for-the-badge&color=248045" />
+                <img alt="GitHub commits"
+                     src="https://img.shields.io/github/commit-activity/t/StartITBot/discord.builders?style=for-the-badge&color=248045" />
+            </div></a>
```

Comment:
I haven't seen these rendered, but I would keep only the star on GitHub. The other two badges just add noise and are not really relevant to a user (I would say put them in the readme, but at that point they are already in GitHub, so they can find that info quickly too)